assets/proxy/main.go

```diff
@@ -93,5 +94,8 @@ var httpClient = &http.Client{
 			Timeout:   10 * time.Second,
 			KeepAlive: 0,
 		}).Dial,
+		TLSClientConfig: &tls.Config{
```

Comment:
Was this change required, or was this added to work around a limitation with your test environment?